torch/distributions/exp_family.py

```diff
@@ -0,0 +1,52 @@
+import torch
+from torch.distributions.distribution import Distribution
+from torch.autograd import Variable
+
+
+class ExponentialFamily(Distribution):
+
+    _zero_carrier_measure = None
```

Comment:
As I understand from the [Bregman divergence paper](https://pdfs.semanticscholar.org/fab3/337bcac9d026bb0e370cc644c46823e574c9.pdf), entropy can be computed if the expected value of the carrier measure is known (`b` in the paper's notation). Could you replace the `_zero_carrier_measure` attribute with an abstract function or property?
```py
class ExponentialFamily(Distribution):
    @property
    def carrier_measure_mean(self):
        raise NotImplementedError

class Normal(ExponentialFamily):
    carrier_measure_mean = 0
```